### ISSUE TITLE
Using Cross Entropy Loss for Classification

### ISSUE THREAD

**Issue by muditchaudhary** [muditchaudhary/RepPoints-x-Libra-R-CNN-x-Transformer-self-attention#4]
Usage of Cross-Entropy Loss requires you to provide a sampler. But in order to do that, we need to modify reppoints_head.py
